src/pybind/mgr/mgr_module.py

```diff
@@ -426,11 +426,14 @@ def call(self,
              mgr: Any,
              cmd_dict: Dict[str, Any],
              inbuf: Optional[str] = None) -> HandleCommandResult:
-        kwargs = self._collect_args_by_argspec(cmd_dict)
-        if inbuf:
-            kwargs['inbuf'] = inbuf
-        assert self.func
-        return self.func(mgr, **kwargs)
+        try:
+            kwargs = self._collect_args_by_argspec(cmd_dict)
+            if inbuf:
+                kwargs['inbuf'] = inbuf
+            assert self.func
+            return self.func(mgr, **kwargs)
+        except ValueError as e:
```

Comment:
Do you want to try and catch AssertionError too?  I'm not sure how common the (new) line 433 would be.

Reply:
I changed the impl to address @adk3798 concerns. Plz take a look to the new changes.